modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/Detail.js

```diff
@@ -41,10 +42,25 @@ export default function Detail({
 	trafficSourcesDataProvider,
 	trafficVolumeDataProvider,
 }) {
+	const featureFlag = true;
+
 	const chartDispatch = useContext(ChartDispatchContext);
 
 	const storeDispatch = useContext(StoreDispatchContext);
 
+	const currentPageMocked = {
+		data: {
+			countrySearchKeywords:
```

Comment:
This data is not showing keywords so I would remove the word "keywords" from its name

Reply:
Done! I was only using it to maintain the structure of the data from the backend